Project/src/MakeCall/Lobby.js

```diff
@@ -31,34 +41,52 @@ const Lobby = ({ call }) => {
         }
     };
 
-    const admitAllParticipants = async () => {
+    capabilitiesChangedHandler = (capabilitiesChangeInfo) => {
+        console.log('lobby:capabilitiesChanged');
+        for (const [key, value] of Object.entries(capabilitiesChangeInfo.newValue)) {
+            if(key === 'manageLobby' && value.reason != 'FeatureNotSupported') {
+                (value.isPresent) ? this.setState({ canManageLobby: true }) : this.setState({ canManageLobby: false });
```

Comment:
Is it possible that this change could go in the other direction? From being able to handle to not handle? If so I think we can't check just .isPresent

Reply:
if the capabilities change from can handle to cannot handle, then isPresent will be false and we set state to false here.  not seeing and issue. Do you see an issue if so?